arch/xtensa/src/esp32s2/Kconfig

```
@@ -683,6 +681,14 @@ endmenu # SPI Flash configuration
 menu "SPI RAM Config"
```

```suggestion
menu "SPI RAM Configuration"
```
To be consistent to the other menus